lib/incito-browser/incito.ts

```diff
@@ -266,7 +266,7 @@ function renderView(view, canLazyload: boolean) {
             const src = String(new URL(view.src));
 
             if (isDefinedStr(view.src)) {
-                if (canLazyload) {
+                if (canLazyload && !enableLazyLoading) {
```

Comment:
The naming is confusing. You would think enableLazyLoading would enable lazyloading of images. But the enableLazyLoading relates to the Incito